detect_ai_content/ml_logic/for_texts/XGB.py

```diff
@@ -0,0 +1,99 @@
+import pandas as pd
+import os
+
+import pickle
+from detect_ai_content.ml_logic.for_texts.create_internal_features import InternalFeatures
+import pdb
+
+class XGB:
```

Comment:
What is the difference between XGB and XGBoost ?

Reply:
No diff, just an abbreviation